.pyinstaller/hooks/hook-sunpy.py

```diff
@@ -0,0 +1,25 @@
+#-----------------------------------------------------------------------------
```

Comment:
Do we need this copyright?

Reply:
We (as in PyInstaller) do to appease GPL. You guys shouldn't because the BSD license doesn't mandate that you put a copyright header on every file. And it's your file anyway so it should be under your copyright anyway. You can (should?) get rid of it here.

Reply:
Thanks @bwoodsend, I think we will remove it and have this file under our BSD licence instead.